src/main/resources/io/viash/runners/nextflow/arguments/_processOutputValues.nf

```diff
@@ -1,12 +1,5 @@
-Map _processOutputValues(Map outputs, Map config, String id, String key) {
+Map _checkValidOutputArgument(Map outputs, Map config, String id, String key) {
```

Comment:
If the function is renamed then the filename should also be renamed, I guess

Reply:
Well, I've added a second function to the file... 🤔